### PR TITLE
Do not apply spacing to news item when no leadimage is visible.

### PR DESCRIPTION
**Pull request by bierik**
Depends on https://github.com/4teamwork/ftw.simplelayout/pull/312

Fixes https://github.com/4teamwork/plonetheme.blueberry/issues/35
Fixes https://github.com/4teamwork/bl.web/commit/9cd25226cd72fbf2aa48496a1924961497d79e4c#commitcomment-16356946

Some examples of bl.web
Newslisting Block:
bildschirmfoto 2016-03-02 um 14 45 08
bildschirmfoto 2016-03-02 um 14 45 39
bildschirmfoto 2016-03-02 um 14 45 57

Newslisting:
bildschirmfoto 2016-03-02 um 14 47 33
bildschirmfoto 2016-03-02 um 14 47 42
bildschirmfoto 2016-03-02 um 14 47 56

Newslisting Portlet:
bildschirmfoto 2016-03-02 um 14 49 27
bildschirmfoto 2016-03-02 um 14 49 39
bildschirmfoto 2016-03-02 um 14 49 53
bildschirmfoto 2016-03-02 um 14 50 04

Newslisting with default image:
bildschirmfoto 2016-03-02 um 15 45 34

Newslistingblock with no leadimage:
bildschirmfoto 2016-03-02 um 15 45 53

### CODE REVIEW

File: ftw/news/browser/resources/news.scss [hunk 1]
```diff
@@ -2,7 +2,10 @@
 @include portal-type-font-awesome-icon(ftw-news-newsfolder, newspaper-o);
 @include portal-type-font-awesome-icon(ftw-news-newslistingblock, newspaper-o);
 
-$leadimage-width: $scale-preview-width;
+$leadimage-width: $scale-news-listing-image-width;
+$leadimage-height: $scale-news-listing-image-height;
+
+$leadimage-max-width: 200px;
```
Comment (jone) [inline ftw/news/browser/resources/news.scss]: `!default` would be nice for all variables here :wink:

File: ftw/news/browser/resources/news.scss [hunk 2]
```diff
@@ -21,18 +24,30 @@ $leadimage-width: $scale-preview-width;
       @include screen-small() {
         float: left;
         > img {
-          max-width: $leadimage-width;
+          max-width: $leadimage-max-width;
         }
       }
+
+      background: url("++resource++ftw.news.resources/newspaper.svg") center / 50% no-repeat;
```
Comment (jone) [inline ftw/news/browser/resources/news.scss]: Please use the `$portal-url` variable as basis for the url on order to make it cacheable.

File: ftw/news/browser/resources/news.scss [hunk 3]
```diff
@@ -21,18 +24,30 @@ $leadimage-width: $scale-preview-width;
...
+      min-width: $leadimage-max-width;
+      min-height: $leadimage-max-width;
+
       > img {
         vertical-align: middle;
       }
+
+      &.no-image {
+        background: url("++resource++ftw.news.resources/newspaper.svg") center / 50% no-repeat;
```
Comment (jone) [inline ftw/news/browser/resources/news.scss]: Please use the `$portal-url` variable as basis for the url on order to make it cacheable.

**Comment by jone** [comment]
:+1: nice!

**Comment by bierik** [comment]
@jone updated

**Pull request merged**